ReactAndroid/src/main/jni/react/JSCExecutor.cpp

```diff
@@ -277,7 +277,8 @@ void JSCExecutor::flush() {
 
   if (!ensureBatchedBridgeObject()) {
     throwJSExecutionException(
-        "Couldn't get the native call queue: bridge configuration isn't available. This shouldn't be possible. Congratulations.");
+        "Couldn't get the native call queue: bridge configuration isn't available. You may need to use \"react-native start\" manually, "
+        "or to run \"adb reverse tcp:8081 tcp:8081\" if you are testing on a physical device.");
```

Comment:
Sorry I'm not too familiar with the project's structure, but I assumed this message could only be shown on Android device, given that the file is under the directory `ReactAndroid`